KlockanAPI/src/KlockanAPI.Domain/Models/Webex/MeetingReport.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KlockanAPI.Domain.Models.Webex
+{
+    public class MeetingReport
+    {
+        public List<ParticipantReport> items { get; set; }
+    }
+}
```

Comment:
please change to 
`
namespace KlockanAPI.Domain.Models.Webex;

public class MeetingReport
{
     public List<ParticipantReport> items { get; set; }
}
`